tmt/beakerlib.py

```diff
@@ -167,6 +193,7 @@ def fetch(self):
                         ['git', 'clone', self.url, directory],
                         env={"GIT_ASKPASS": "echo"})
                 else:
+                    assert self.path
```

Comment:
I'd propose `assert self.url or self.path` before line 191, to signal that at least one of them shall be set, and it should produce consistent.

Reply:
so add another assert @happz ? I need to keep this here otherwise it is not happy with just that at all

Reply:
added rather if / else